front/components/agent_builder/transformAgentConfiguration.ts

```diff
@@ -204,11 +204,12 @@ export function getDefaultAgentFormData(user: UserType): AgentBuilderFormData {
     instructions: "",
     generationSettings: {
       modelSettings: {
-        modelId: GPT_4O_MODEL_CONFIG.modelId,
-        providerId: GPT_4O_MODEL_CONFIG.providerId,
+        modelId: CLAUDE_4_SONNET_DEFAULT_MODEL_CONFIG.modelId,
+        providerId: CLAUDE_4_SONNET_DEFAULT_MODEL_CONFIG.providerId,
       },
       temperature: 0.7,
-      reasoningEffort: GPT_4O_MODEL_CONFIG.defaultReasoningEffort || "none",
+      reasoningEffort:
```

Comment:
Do we need this? Seems like we don't have any option so I assume we can remove it for sonnet 4? 

<img width="610" height="259" alt="Image" src="https://github.com/user-attachments/assets/97e814ef-f34f-4bd0-9940-fea837f1f334" />

Reply:
Made it configurable iif there's more than one possible reasoning effort